docs/rest_api.md

```diff
@@ -8,147 +8,3 @@ When MMS startup, it start two web services:
 
 By default, MMS listening on 8080 port for Inference API and 8081 on Management API.
 Both API is only accessible from localhost. Please see [MMS Configuration](configuration.md) for how to enable access from remote host. 
```

Comment:
Seems like we removed a section here. Don't we need reference to the inference api doc here?

Reply:
This section was from 0.4 and didn't clean up.

In this file we already have links to inference_api.md and management_api.md.

Reply:
Ok. Saw the examples in that removed section and thought you were removing the current examples.